test/gtest/ucp/test_ucp_am.cc

```diff
@@ -1896,19 +1896,23 @@ class test_ucp_am_nbx_rndv_memtype : public test_ucp_am_nbx_rndv {
         return test_ucp_am_nbx_eager_memtype::get_test_variants(variants);
     }
 
-    void init()
+    void init() override
     {
-        modify_config("RNDV_THRESH", "128");
         test_ucp_am_nbx::init();
     }
 
 private:
-    virtual ucs_memory_type_t tx_memtype() const
+    unsigned get_send_flag() const override
+    {
+        return test_ucp_am_nbx_rndv::get_send_flag() | UCP_AM_SEND_FLAG_RNDV;
```

Comment:
maybe set this flag in `test_ucp_am_nbx_rndv` instead and remove setting `RNDV_THRESH` from its constructor?

Reply:
it may invalidate some tests that check for the specific threshold - for example rndv_flag_zero_send, just_header_rndv, header_and_data_rndv